functional-tests/devscripts/prepare_env.sh

```diff
@@ -2,9 +2,9 @@
 
 set -e
 echo "Installing docker..."
-yum install -y docker
+yum install -y docker | cat # Suppress multiple-line output for package installation
```

Comment:
I'd clarify that piping to `cat` is to suppress tty output (e.g. progress). There is also the `-q` option to suppress all output.

Reply:
I don't want to silence it entirely. I just want to avoid it adding 3 new lines into logs every time the progress bar moves. this way it only writes the install log for every package once